### ISSUE TITLE
Admin user: Flag court order blocks as likely mistakes

### ISSUE THREAD

**Issue by JimKillock** [openrightsgroup/blocked-org-uk#291]
Allow an admin user to flag court order blocks as a Likely incorrect block with options:

* Domain inactive
* Domain for sale or parked
* Domain in legitimate usage
* Domain currently broken
* Domain does not resolve
* Domain may be abusive

Include options for abuse
* click fraud / redirects
* malware

Include
* field to describe the error
* date the error was observed

Additionally, timeout errors could be used to auto-flag as likely incorrect

From #299 – allow admin user to flag URLs as:
* duplicates (eg Capitalised)

**Comment by dantheta** [comment]
I've kept the URL admin in the control panel, as the frontend can't normally look up URLs that have case differences or domain with a '/'.

**Comment by JimKillock** [comment]
See also #319

**Comment by JimKillock** [comment]
Hi @dantheta – note to say @gwire would like to input this information so it would be great if this could move to the top of the todo list :)

**Comment by JimKillock** [comment]
Hi @dantheta: we have campaigning to do that relies on us using this tool so we need this ASAP.

**Comment by dantheta** [comment]
The court order view now has a flag link by each listed URL, which leads to a form for recording the details.

**Comment by dantheta** [comment]
The court order view now has a flag link by each listed URL, which leads to a form for recording the details.

**Comment by JimKillock** [comment]
Thanks @dantheta – could you for convenience add an active link (open in new tab) for the links on the list, (could be a button or whole URL as a link)  to aid us going through the checking process?

And same for the "Flag http://couchtuner.bz" form pages.

**Comment by JimKillock** [comment]
Also: for the date field, could it pre-fill today's date?

**Comment by dantheta** [comment]
All done.  I've tried reviewing a few, and it seems to go through pretty smoothly.

**Comment by JimKillock** [comment]
Great :) Now we will need to work out a way to display all this on the front end.

As a first step, we could add the per-URL flag information on the legal blocks page, and the site report page.

On the site report page: the flag information should be placed underneath 
"This site has been blocked by court order."
in a smaller text eg: <p>This site has been flagged as incorrectly blocked by as it is no longer serving content from a website subject to court order. It has been recorded as XXX."</p>

On the legal blocks page:

Place the same information at the foot of the table info for the URL, above the thin line.

(Once we have all the sites reviewed, we can create some summary statistical information.)

**Comment by JimKillock** [comment]
feedback: 

"It looks like some of the flags aren't showing up on the main page, which registers only 30 'incorrectly blocked' sites. https://www.blocked.org.uk/legal-blocks/errors?sort=created "

**Comment by JimKillock** [comment]
This I think is done and can be closed
